<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaureWidget</class>
 <widget class="QWidget" name="LaureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="GlobalGridLayout_">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="5">
        <widget class="LeechCraft::Laure::Player" name="Player_">
         <property name="styleSheet">
          <string notr="true">padding: 25px; background: black;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="CommandFrame_">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>40</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="PositionSlider_">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="LeechCraft::Laure::VolumeSlider" name="VolumeSlider_">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Volume slider</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="TimeStamp_">
         <property name="text">
          <string>[00:00:00/00:00:00]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="LeechCraft::Laure::PlayListWidget" name="PlayListWidget_" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::Laure::Player</class>
   <extends>QFrame</extends>
   <header>player.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeechCraft::Laure::PlayListWidget</class>
   <extends>QWidget</extends>
   <header>playlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeechCraft::Laure::VolumeSlider</class>
   <extends>QSlider</extends>
   <header>volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
